app/Models/Sprint.php

```diff
@@ -81,4 +82,16 @@ public function getLastUpdatedDate(): Carbon
     {
         return $this->lastUpdated;
     }
+
+    public function containsToday(): bool
+    {
+        if (empty($this->getStartDate()) || $this->getEndDate()) {
+            return false;
+        }
```

Comment:
I think the '||' check has to be explicit on both side.
```suggestion php
        if (empty($this->getStartDate()) || empty($this->getEndDate())) {
            return false;
        }
```

Reply:
Also, this is super nitpicky of me, but can we do the else and both brackets on the same line like
```php
        } else {
````

Reply:
Updating this to:

```php
    public function containsToday(): bool
    {
        $period = $this->sprintPeriod();
        return !empty($period) ? $period->contains(Carbon::now()) : false;
    }

    public function sprintPeriod(): ?CarbonPeriod
    {
        if (empty($this->getStartDate()) || empty($this->getEndDate())) {
            return null;
        }
        return CarbonPeriod::create($this->getStartDate(), $this->getEndDate());
    }
```

This should resolve both issues, but please feel free to review again!